src/averageAges.js

```diff
@@ -14,12 +14,19 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+function calculateMenAverageAge(people, century = null) {
```

Comment:
The second argument 'century' is expected to be a number. Using 'null' as default value may lead to incorrect results when it is not provided. Consider using an appropriate number or 'undefined' as default value.